### DISCUSSION TITLE
[Support] 可以增加一下knex组件吗？

### DISCUSSION THREAD

**Discussion by hooliy** [midwayjs/midway#3864] [Q&A]
### Describe the problem（描述问题）

用MikroORM太麻烦，可以直接封装一个knexjs包吗？

### Midway Versions（Midway 版本）

latest

**Comment by waitingsong** [comment]
看看这里的 kmore 组件（基于 knex 封装）是否满足你的需求 https://midwayjs.org/docs/awesome_midway
